docs/slides/content/tech-exercise-ii.md

```diff
@@ -247,27 +246,29 @@ Choose your own adventure but both half of the teams must be successful to finis
 
 
 #### Feedback
-* Q&A
+* Q & A
 * Real World Stories
 * Pull Requests Welcome 🦄
 
 
 
 #### WHO - Pipelines <!-- .element: class="title-bottom-left" -->
 <!-- .slide: data-background-size="contain" data-background-image="images/tech-exercise-ii/example-who.png", class="black-style" data-background-opacity="1"	 -->
+<!--Tech Edit - this slide is empty except for the heading. -->
 
 
 
 #### WHO - App of Apps <!-- .element: class="title-bottom-left" -->
 <!-- .slide: data-background-size="contain" data-background-image="images/tech-exercise-ii/example-who.png", class="black-style" data-background-opacity="1"	 -->
+<!--Tech Edit - this slide is empty except for the heading. -->
 
 
 
 ### Summary
-* **Continuous Integration**: Integrate several changes into a shared repo and rely heavily on testing to make sure that the changes and commits works and have passed all (unit testing / code quality / code conduct).
-* **Continuous Delivery**: The goal is to have a working artifact that's ready to be deployed at any time. It relies on testing and automation to make sure that it has passed all functional tests and can be Manually deployed (push button).
-* **Continuous Deployment/Release**: One step further now that we are sure that this is ready to deploy to production then we can rely on features toggle, dark launches or other deployment strategy to promote to other critical environments asap using these techniques for the new features/fixes won't risk the operation.
-
+* **Continuous Integration**: Integrate several changes into a shared repository and rely heavily on testing to make sure that the changes and commits work and have passed all tests (unit testing / code quality / code conduct).
+* **Continuous Delivery**: The goal is to have a working artifact that's ready to be deployed at any time. It relies on testing and automation to make sure that it has passed all functional tests and can be manually deployed (push button).
+* **Continuous Deployment/Release**: A further step, now that we are sure this artifact is ready to deploy to production, is to rely on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.
```

Comment:
```suggestion
* **Continuous Deployment/Release**: A further step, now that we are sure this artifact is ready to deploy to production, we can now rely on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.
```

Reply:
The grammar is not quite right in the suggestion. If we leave out the clause set off by commas, we have "A further step, ..., now we can..." which isn't correct.